bolt/lib/nonlinear_solver/tests/test_compute_moments.py

```diff
@@ -33,6 +33,17 @@
                      energy=[1, 1, 1]
                      )
 
+@af.broadcast
+def maxwell_boltzmann(rho, T, p1_b, p2_b, p3_b, p1, p2, p3):
```

Comment:
What is `@af.broadcast` / what does it do and to what functions should that be added to at the user level?